src/main/java/algorithms/medium/HouseRobberII.java

```diff
@@ -0,0 +1,25 @@
+package algorithms.medium;
+
+public class HouseRobberII {
+
+    public int rob(int[] nums) {
+
```

Comment:
```suggestion
        if (nums.length == 0)
            return 0;

        if (nums.length == 1)
            return nums[0];
```

Reply:
We don't need the 0-check. The 1-check is needed though.